airbyte-integrations/connectors/source-tiktok-marketing/source_tiktok_marketing/schemas/basic_reports.json

```diff
@@ -0,0 +1,142 @@
+{
+  "type": ["null", "object"],
+  "properties": {
+    "metrics": {
+      "type": ["null", "object"],
+      "properties": {
+        "campaign_name": {
+          "type": ["null", "string"]
+        },
+        "campaign_id": {
+          "type": ["null", "number"]
```

Comment:
are these `id` columns really number? not integer? (they'll have a float type instead of int)

Reply:
fixed